content/docs/join_us/recruitment_FAQs.md

```diff
@@ -74,17 +95,43 @@ For more information about the Institute's reasonable adjustment policy see [the
+
+### I'm a woman / person of colour / LGBTQIA+ / person with disabilities / part of an underrepresented group(s). Will I feel welcome and supported in the team?
+
+**We'd love to have you join our team!**
+Inclusivity and diversity aren't just things we encourage because we're beholden to some corporate criteria; we believe they are *essential* to good research.
+We'd really encourage you to apply!
 
-This role presents a new exciting opportunity to join the growing team of research computing experts at The Turing. The team assists the Turing community with designing, building, deploying, and maintaining research infrastructures, as well as providing service delivery and user support for the Institute’s research computing platforms. This is an opportunity for a technically-skilled professional with a good understanding of either cloud or high-performance computing (HPC), experience with both application and infrastructure, and an interest in facilitating and enhancing world-leading data science and AI research.
-You will build tools and automation to eliminate mundane operations and craft repeatable processes. As part of the team, you will also work with other research software engineers and data scientists on various projects and will participate in all phases of cloud / HPC development, from requirements-capture, definition, and design; to development, deployment, and maintenance; to performance tuning and monitoring. You are also expected to keep up-to-date with the latest cutting-edge developments, and/or make use of the UK’s most advanced computing facilities like Dawn, ARCHER 2, JADE 2, and Baskerville.
+Our team has a very motivated group of people that work on things such as:
+
+- inputting to institute-wide policies (e.g. trans and non-binary inclusion)
+- doing analysis of diversity data, like for our [Gender Pay Gap Report](https://www.turing.ac.uk/sites/default/files/2023-03/2023.03.30_gender_pay_gap_report.pdf)
+- making job ads more inclusive (not closing applications early, publishing salary bands, using clear language)
+- being ambassadors for REG and the Turing at events like the [Hopper Colloquium](https://www.ucl.ac.uk/computer-science/about/equity-diversity-and-inclusion/gender-equality-athena-swan/london-hopper-colloquium) and the [BCSWomen Lovelace event](https://bcswomenlovelace.bcs.org/)
+- Attending network groups – a place to find community, organise events, and work on issues specific to that network. They exist for
+    - LGBTQIA+
+    - Race equality
+    - Gender equality
+    - Disability and wellbeing
+- and much more!
+
+You can read more about a sample of the Equality, Diversity and Inclusion (EDI) initiatives in REG and the Turing on [this page]({{< relref "docs/working_at_the_turing/edi" >}}).
+Also, if you would like to speak to somebody in REG (in confidence) about this before applying, we would completely welcome that.
```

Comment:
I hesitate to promise that 'people you speak to won't be involved in your application', because that was not my experience when I applied to REG. @IFenton Do you think this can definitely be arranged? If you're confident, please feel free to edit this bit

Reply:
I've updated the wording to clarify this